docs/the_black_code_style/future_style.md

```diff
@@ -34,6 +34,17 @@ with \
 Although when the target version is Python 3.9 or higher, _Black_ will use parentheses
 instead since they're allowed in Python 3.9 and higher.
 
+An alternative to consider if the backslashes in the above formatting is undesirable is to use [`contextlib.ExitStack`](https://docs.python.org/3.9/library/contextlib.html#contextlib.ExitStack) to combine context managers in the following way:
```

Comment:
```suggestion
An alternative to consider if the backslashes in the above formatting is undesirable is to use {external:py:obj}`contextlib.ExitStack` to combine context managers in the following way:
```
Let's use intersphinx to avoid hardcoding the URL.

Reply:
Love it, and I didn't know that was a thing.  Thank you!

Reply:
Yeah [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) is pretty awesome. We currently only have it set up to pull down the object inventory file[^1] for CPython's docs, but that's already useful :)

[^1]: If you didn't know, Sphinx exports a file that contains all of the "objects" like documents, headers, and even API functions, classes, and names if configured to do so.